README.md

```diff
@@ -7,8 +7,48 @@ openSSL module as extension for chassis.io. See https://github.com/Chassis/Chass
 1. Run `vagrant reload --provision` or just `vagrant up`
 1. Profit.
 
-## Caveats and known problems:
+## Notes:
 
-1. I haven't tested with multisite.
-1. You also need to modify the `wp-config.php` in the root directory and modify the `WP_SITEURL` and `WP_HOME` constants to use https instead of http
-1. You'll have to add the `.cert` file in the root directory of Chassis to your keychain and set it to "Always trust" (right click on it, Get Info) to avoid the red "your connection is not encrypted" message. Don't know how to do it on a Windows machine. Let me know :)
+1. You also need to modify the `wp-config.php` in the root directory and modify the `WP_SITEURL` and `WP_HOME` constants to use https instead of http.
+1. Both the certificate and key are exported to `chassis/` directory, so it can be used with local dev servers, like webpack dev server.
+
+## Trusting the certificate
+
+In order to avoid security errors and get that nice green padlock in your location bar, you should add the site's SSL certificate to your trust store. The certificate can be found at chassis/{DOMAIN_HERE}.cert .
+
+### macOS via CLI
+`sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain chassis/{DOMAIN_HERE}`
+
+### Firefox on any operating system:
```

Comment:
I also found this method for Firefox:

https://github.com/humanmade/altis-local-chassis/tree/master/docs#firefox

- Open Firefox
- Browse to about:config
- Set `security.enterprise_roots.enabled` to true

Reply:
Added as alternative method.